tests/cmd/extract-list/extract-list.rv64-qemu-virt.0.stdout

```diff
@@ -0,0 +1,22 @@
+        SIZE NAME
```

Comment:
I might've missed this on an earlier comment - do we actually want to check these in, or should they be .gitignored?